Tests/scripts/common.py

```diff
@@ -110,8 +109,7 @@ def calculate_results_table(jira_tickets_for_result: dict[str, Issue],
     total_row: list[Any] = ([NOT_AVAILABLE] * fixed_headers_length + [TestSuiteStatistics()
                                                                       for _ in range(len(server_versions_list))])
     total_errors = 0
-    for result, result_test_suites in tqdm(results.items(), desc="Generating summary table", unit="result",
-                                           leave=True, colour='green', miniters=10, mininterval=5.0):
+    for result, result_test_suites in results.items():
```

Comment:
tqdm is a progress bar, no need for it now as the logic has changed and the it complets very fast.